examples/typescript-reactflux/.gitignore

```diff
@@ -0,0 +1,19 @@
+# Generated folders 
+build/
+node_modules/
```

Comment:
don't you use node_modules for front-end dependencies, then? That's how all other example have been doing it, is there something specific preventing it?